DiscordChatExporter.Core/Exporting/MessageExporter.cs

```diff
@@ -70,7 +70,15 @@ public async ValueTask ExportMessageAsync(
         MessagesExported++;
     }
 
-    public async ValueTask DisposeAsync() => await ResetWriterAsync();
+    protected async ValueTask EnsureFileIsCreated(CancellationToken cancellationToken = default) {
+        // causes the file to be created whether there were messages written or not
+        await GetWriterAsync(cancellationToken);
+    }
```

Comment:
It's better to inline this method (call `GetWriterAsync` directly) into `DisposeAsync`

Reply:
The intention is more clear this way, but your call; done in https://github.com/Tyrrrz/DiscordChatExporter/pull/1349/commits/daff360ce4cf0cc7c6b9555559d0fe3da17cefe3